packages/module/patternfly-docs/content/extensions/virtual-assistant/examples/VirtualAssistant/VirtualAssistantWithActions.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import VirtualAssistant from '@patternfly/virtual-assistant/dist/dynamic/VirtualAssistant';
+// import VirtualAssistantAction from '@patternfly/virtual-assistant/dist/cjs/VirtualAssistantAction';
+
+export const VirtualAssistantWithActions: React.FunctionComponent = () => (
+  <VirtualAssistant actions={<>
+
+  </>} />
+);
+
+/*
+    <VirtualAssistantAction>
+      <AngleDownIcon/>
+    </VirtualAssistantAction>
```

Comment:
@fhlavac I'm having some issues here.

![image](https://github.com/patternfly/virtual-assistant/assets/3845764/cb590bb3-8963-44a0-b139-4f2d0d5bdb64)

Looks like it can't find `VirtualAssistantAction` - the default import exists. A similar issue happens if I try to import 

```
import { AngleDownIcon } from '@patternfly/react-icons';
```

Have you seen this behaviour before?  I'm guessing it might be a config i'm missing.

Reply:
cc @fhlavac

Reply:
![image](https://github.com/patternfly/virtual-assistant/assets/50696716/2440a4d0-0c98-4600-be8a-5b34755cbd44)
Adding the imports to the .md file as well should help

Reply:
Thanks! I missed that part :sweat_smile: